package.json

```diff
@@ -1,65 +1,21 @@
 {
-  "name": "@supabase/supabase-auth-helpers",
+  "name": "@supabase/auth-helpers",
   "version": "0.0.0",
-  "description": "A collection of framework specific Auth utilities for working with Supabase.",
-  "engines": {
-    "node": "^10.13.0 || >=12.0.0"
-  },
+  "private": true,
   "scripts": {
-    "clean": "rimraf dist",
-    "pretty": "prettier --write \"src/**/*.{ts,tsx}\" \"src/*.ts\"",
-    "lint": "tsc --noEmit && eslint --fix --ext .ts ./src",
-    "docs": "typedoc --options typedoc.js src",
```

Comment:
We should be able to keep the typedoc stuff and just change the command to `"docs": "typedoc --options typedoc.js packages"`, no?

Reply:
Is this on a per package basis? where will this be used?

Reply:
It runs on merge (https://github.com/supabase-community/supabase-auth-helpers/blob/next/.github/workflows/docs.yml) to publish https://supabase-community.github.io/supabase-auth-helpers/

Reply:
Ok this is a bit more tricky to get working in a monorepo. I've seen a few examples using it but I'm still trying to figure out how to get it working in our setup.